### ISSUE TITLE
cann't install via windows

### ISSUE THREAD

**Issue by xiangxiang0826** [diambra/arena#77]
env:
win10
python 3.9
pip 23.1.2

 error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in gym setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
      [end of output]

I have tried change python 3.9 to python 3.11, and pip install --user --upgrade setuptools . It still doesn't work.

**Comment by alexpalms** [comment]
Hi @xiangxiang0826, I assume this is the result of 

```
pip install diambra
pip install diambra-arena
```
Right? If yes, this seems to be related with a recent change with pip. We will move from `setup.py` to `pyproject.toml` soon, but for now the workaround that should work is installing a specif wheel version before installing diambra:
```
pip install wheel==0.38.4
pip install diambra
pip install diambra-arena
```

Can you try that and let us know if this solved the problem?

**Comment by alexpalms** [comment]
@xiangxiang0826 we just found out that another python package has conflicts with the current dependency structure, so instead of pinning `wheel` you need, for now, to pn `setuptools`, so this is the workaround to install diambra:

1) Create a containerized environment using (mini)conda and a given python version (e.g. 3.9):
`conda create -n diambra python=3.9`
2) Install `diambra-arena` as follows:
```
pip install setuptools==66.0.0
pip install diambra
pip install diambra-arena
```
This should fix your problem. 

We will be releasing an updated version soon that should integrate this fix, and I will keep this issue open until then.

**Comment by alexpalms** [comment]
@xiangxiang0826 this is just to confirm that we fixed this issue and just released a new `diambra-arena` version `2.1.0rc12` (https://pypi.org/project/diambra-arena/), so you should now be able to properly install the package with the standard commands:

```
pip install diambra
pip install diambra-arena
```

This closes this issue, but do not hesitate to write again in case you need additional support
